README.md

```diff
@@ -49,8 +49,7 @@
 
 ## Overview
 
-Cardano Wallet helps you manage your Ada. You can use it to send and
-receive payments on the [Cardano](https://www.cardano.org) blockchain.
+Cardano Wallet is a software that helps you manage your Ada. You can use it to send and receive payments on the [Cardano blockchain](https://www.cardano.org).
```

Comment:
Since "software" is an uncountable noun, it sounds unnatural to say "a software".

But we can say "a software application", since "applications" are countable:

```suggestion
Cardano Wallet is a software application that helps you manage your Ada. You can use it to send and receive payments on the [Cardano blockchain](https://www.cardano.org).
```

Reply:
I will removed the article `a`.